docs/content/latest/yugabyte-platform/create-deployments/async-replication-platform.md

```diff
@@ -11,12 +11,110 @@ menu:
 aliases:
 type: page
 isTocNested: true
-showAsideToc: false
+showAsideToc: true
 ---
 
-You can perform deployment via unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between data centers.
+Yugabyte Platform allows you to use its UI or API to manage asynchronous replication between independent YugabyteDB clusters. You can perform deployment via unidirectional (master-follower) or bidirectional (multi-master) asynchronous replication between two data centers.
```

Comment:
where is bidirectional covered?

Reply:
As per Jason, "There is no easy / separate way to do this from the platform (via UI or API). The "expected" / "supported" way to set up bi-directional replication (via UI or API) is to create two separate replication configurations (one for universe A->B, another for universe B->A)"

Reply:
I'll mention it. Will also add a link to the asynch replication doc in the the core db.

Reply:
Added a small section:

Setting Up Bidirectional Replication

You can set up bidirectional replication using either the Yugabyte Platform UI or API by creating two separate replication configurations. Under this scenario, a source universe of the first replication becomes the target universe of the second replication, and vice versa.